### PR TITLE
Rails Sprinkles: Consistent naming for SPA

### PR DESCRIPTION
**Pull request by Kacegz**
## Because

Naming for non-abbreviated SPA isn't consistent.
https://github.com/TheOdinProject/curriculum/pull/26178#issuecomment-1688009136

## This PR

- Change ``Single Page Application`` and ``single page application`` to ``Single-page application`` in [Turbo](https://www.theodinproject.com/lessons/ruby-on-rails-turbo) lesson

## Issue

Related to #26178

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: ruby_on_rails/rails_sprinkles/turbo.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 
-That's where the concept of a **single page application** comes in. This section will be about how we can use a framework known as **Turbo** to implement single page application behaviour in our very own Rails application. This section covers a lot of tools and you may not fully understand them as you read through the first time. That's okay, consider this lesson to be a resource that you can refer back to when you actually begin using the  tools mentioned and get more of a visual as to what exactly is going on in your views.
+That's where the concept of a **Single-page application** comes in. This section will be about how we can use a framework known as **Turbo** to implement Single-page application behaviour in our very own Rails application. This section covers a lot of tools and you may not fully understand them as you read through the first time. That's okay, consider this lesson to be a resource that you can refer back to when you actually begin using the  tools mentioned and get more of a visual as to what exactly is going on in your views.
```
Comment (ManonLef) [inline ruby_on_rails/rails_sprinkles/turbo.md:5]:
```suggestion
That's where the concept of a **single-page application** comes in. This section will be about how we can use a framework known as **Turbo** to implement single-page application behaviour in our very own Rails application. This section covers a lot of tools and you may not fully understand them as you read through the first time. That's okay, consider this lesson to be a resource that you can refer back to when you actually begin using the  tools mentioned and get more of a visual as to what exactly is going on in your views.
```

File: ruby_on_rails/rails_sprinkles/turbo.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ This section contains a general overview of topics that you will learn in this l
 - How to use Turbo Streams to update your pages.
 - What Turbo Native is used for.
 
-### Single Page Applications (SPAs)
+### Single-page applications (SPAs)
 
-Before getting into Turbo, it's important to understand the concept of a Single Page Application, or SPA. A SPA is an
+Before getting into Turbo, it's important to understand the concept of a Single-page application, or SPA. A SPA is an
```
Comment (ManonLef) [inline ruby_on_rails/rails_sprinkles/turbo.md:20]:
```suggestion
Before getting into Turbo, it's important to understand the concept of a single-page application, or SPA. A SPA is an
```

File: ruby_on_rails/rails_sprinkles/turbo.md [hunk 3]
```diff
@@ -319,9 +319,9 @@ The final piece of Turbo is something that you don't need to know much about for
 
 <div class="lesson-content__panel" markdown="1">
 
-#### Single Page Applications
+#### Single-page applications
 
-1. Read sections 1, 2, and 3 of Bloomreach's [What Is A Single Page Application](https://www.bloomreach.com/en/blog/2018/07/what-is-a-single-page-application.html#whatssingle-page-application)
+1. Read sections 1, 2, and 3 of Bloomreach's [What Is A Single-page application](https://www.bloomreach.com/en/blog/2018/07/what-is-a-single-page-application.html#whatssingle-page-application)
```
Comment (ManonLef) [inline ruby_on_rails/rails_sprinkles/turbo.md:324]:
```suggestion
1. Read sections 1, 2, and 3 of Bloomreach's [What Are Single Page Applications and Why Do People Like Them So Much?](https://www.bloomreach.com/en/blog/2018/07/what-is-a-single-page-application.html#whatssingle-page-application)
```

Changed to correct title of the link

File: ruby_on_rails/rails_sprinkles/turbo.md [hunk 4]
```diff
@@ -355,7 +355,7 @@ The final piece of Turbo is something that you don't need to know much about for
 <details markdown="block">
   <summary>What does SPA stand for & what is it?</summary>
 
-  *   A SPA is a Single Page Application.
+  *   A SPA is a Single-page application.
```
Comment (ManonLef) [inline ruby_on_rails/rails_sprinkles/turbo.md:358]:
```suggestion
  *   A SPA is a single-page application.
```

**Review by ManonLef — CHANGES_REQUESTED** [review]
I'm sorry if I've caused confusion. I think It's best to only capitalise it in headers, and leave it lowercase when used in a sentence inside the paragraphs. 

I've added the changes and you can feel free to commit them.

**Comment by Kacegz** [comment]
Yeah makes sense, my bad
Sorry for additional work

**Comment by ManonLef** [comment]
No don't be sorry! We both had to figure this out 😃 
Thanks so much for taking this on!

**Review by ManonLef — APPROVED** [review]

**Comment by Kacegz** [comment]
Thank you too! 😄

**Pull request merged**